src/webVitals/index.ts

```diff
@@ -1,10 +1,24 @@
 import { usePathname } from 'next/navigation';
 import { useReportWebVitals as useNextReportWebVitals } from 'next/web-vitals';
 import { reportWebVitalsWithPath } from './webVitals';
+import { useCallback, useRef } from 'react';
 export { type WebVitalsMetric } from './webVitals';
-export { AxiomWebVitals } from './components'
+export { AxiomWebVitals } from './components';
 
 export function useReportWebVitals(path?: string) {
-    const pathName = usePathname();
-    useNextReportWebVitals((metric) => reportWebVitalsWithPath(metric, path || pathName));
+  const pathName = usePathname();
```

Comment:
hi @gabrielelpidio , can you elaborate a little bit on what's going on here. What is the problem and what does the fix do? perhaps couple of lines for documentation would be great.

Reply:
Added some documentation around this inside the code